net/DescriptionViewController.m

```diff
@@ -90,12 +121,12 @@ -(void)viewDidLoad
 
 -(void)setDataToView
 {
-    self.titleLabel.text = self.currentIssue.name;
+    self.titleLabel.text = [CurrentItems sharedItems].issue.name;
```

Comment:
It is better to have CurrentItems *currentItems = [CurrentItems sharedItems]; at the beginning of method. The you can use it as self.titleLabel.text = currentItems.issue.name; on every place where you need it